credsweeper/filters/group/credit_card_number_sequense.py

```diff
@@ -11,5 +11,5 @@ class CreditCardNumberSequence(Group):
     """
 
     def __init__(self, config: Config) -> None:
-        super().__init__(config, GroupType.PATTERN)
+        super().__init__(config, GroupType.DEFAULT)
```

Comment:
I wonder why CreditCardNumberSequence` class is `DEFAULT` group.
Please let me know.

And could you please rename the file name? (typo)
`credit_card_number_sequense.py` -> `credit_card_number_sequence.py`

Reply:
Like in previous - DEFAULT means that no predefined filters in super class are assigned.